sdks/nodejs/src/agonesSDK.js

```diff
@@ -93,14 +93,27 @@ class AgonesSDK {
 		});
 	}
 
-	health() {
+	health(errorCallback) {
 		if (this.healthStream === undefined) {
 			this.healthStream = this.client.health(() => {
 				// Ignore error as this can't be caught
 			});
+			if (typeof this.healthStream.on === 'function') {
```

Comment:
@XiaNi I'm just wondering a couple of things about this change:
1. Do we need to listen to the event if we are already passing a callback above? Or is this just to make testing easier?
2. If so, do we need the `typeof` check? Generally I would trust gRPC client types not to change unless we intentionally update the client and then would change this code too

Reply:
Thanks for the detailed explanation. On point 2 I'd add an 'on' to the stream spy object so we can remove this check